pkg/workspace/validate.go

```diff
@@ -53,7 +54,8 @@ func ValidateBindings(ctx context.Context, decls []v1beta1.WorkspaceDeclaration,
 		}
 	}
 	for _, bind := range binds {
```

Comment:
Let's add a comment that mentions why we are checking for whether alpha fields are enabled or not